library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1071,6 +1099,8 @@ public void onResume() {
             // and not a permanent change because we have other uses for warming GPS and in the widget already
             resetRejectionDialogContent();
         }
+
+        this.resumeTrackingService(getApplicationContext());
```

Comment:
This causes other activities to crash with the following stack trace

```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int io.ona.kujaku.services.configurations.TrackingServiceUIConfiguration.getLayoutWidth()' on a null object reference
        at io.ona.kujaku.views.KujakuMapView.initTrackingServiceIcon(KujakuMapView.java:1501)
        at io.ona.kujaku.views.KujakuMapView.resumeTrackingService(KujakuMapView.java:1372)
        at io.ona.kujaku.views.KujakuMapView.onResume(KujakuMapView.java:1103)
        at io.ona.kujaku.sample.activities.FeatureClickListenerActivity.onResume(FeatureClickListenerActivity.java:74)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1277)
        at android.app.Activity.performResume(Activity.java:7094)
```

Reply:
ok sorry. i'm fixing it right now